### PR TITLE
Shift page breaks

### PR DESCRIPTION
**Pull request by b0bi79**
#153

### CODE REVIEW

File: ClosedXML_Tests/Excel/PageSetup/PageBreaksTests.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            var wb = new XLWorkbook();
+            IXLWorksheet sheet = wb.AddWorksheet("Sheet1");
+
+            sheet.PageSetup.AddHorizontalPageBreak(10); 
+            sheet.Row(15).InsertRowsAbove(1);
+            Assert.AreEqual(10, sheet.PageSetup.RowBreaks[0]);
+        }        
+
+        [Test]
+        public void RowBreaksShiftWhenInsertedColumn1()
```
Comment (igitur) [inline ClosedXML_Tests/Excel/PageSetup/PageBreaksTests.cs]: Meant to be 
```c#
public void ColumnBreaksShiftWhenInsertedColumn()
``` 
?

File: ClosedXML_Tests/Excel/PageSetup/PageBreaksTests.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            var wb = new XLWorkbook();
+            IXLWorksheet sheet = wb.AddWorksheet("Sheet1");
+
+            sheet.PageSetup.AddVerticalPageBreak(10); 
+            sheet.Column(5).InsertColumnsBefore(1);
+            Assert.AreEqual(11, sheet.PageSetup.ColumnBreaks[0]);
+        }        
+
+        [Test]
+        public void RowBreaksNotShiftWhenInsertedColumn()
```
Comment (igitur) [inline ClosedXML_Tests/Excel/PageSetup/PageBreaksTests.cs]: Meant to be 
```c#
public void ColumnBreaksNotShiftWhenInsertedColumn()
``` 
?

**Comment by b0bi79** [comment]
Oh, sorry! Of course. Thanks.

**Comment by b0bi79** [comment]
Page breaks should be sorted. Necessarily.

**Pull request merged**